source/javascripts/services/_semverService.js.erb

```diff
@@ -71,10 +84,8 @@ angular.module("BitriseWorkflowEditor").factory("semverService", function() {
             })
             .flatten()
             .tap(function(wVersions) {
-                if (step.version) {
-                    // adding step own version if any
-                    wVersions.push(step.version);
-                }
+                // adding step own version
+                wVersions.push(step.version);
```

Comment:
As discussed with Gergo we should stop adding explicit latest version AKA: `null` but we still need to display whether it's set to latest in yml. So had to add step version here regardless.